### PR TITLE
Move donation information to the top

### PR DESCRIPTION
**Pull request by nalzok**
People should be able to make a donation without reading the whole post or scrolling all the way down. Let's get straight to the point.

<s>By the way, it might be a good idea to add a PayPal link or something similar, so that we can make some one-time donations.</s> Actually I take that back: PayPal may suspend your account simply for the suspicion of money laundering, and even GitHub Sponsor stopped supporting PayPal recently. Still, I would appreciate an option for making a one-time donation.

### CODE REVIEW

**Comment by zloirock** [comment]
Thanks. I agree that this should be brought up, but the text before the quotations can be discouraging reading. I'll think for some minutes.

**Comment by zloirock** [comment]
Could you copy it to https://github.com/zloirock/core-js/blob/master/docs/2023-02-14-so-whats-next.md ?

**Comment by nalzok** [comment]
Sure, it has been done!

File: docs/2023-02-14-so-whats-next.md [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+
+</div>
+
+> If you are looking for `core-js` documentation, it's temporarily moved [here](https://github.com/zloirock/core-js/blob/master/ORIGINAL_README.md).
+>
+> Later, this post will be available [by this link](https://github.com/zloirock/core-js/blob/master/docs/2023-02-14-so-whats-next.md).
```
Comment (zloirock) [inline docs/2023-02-14-so-whats-next.md:11]: Nope, without the rest changes -)

File: docs/2023-02-14-so-whats-next.md [hunk 2]
```diff
@@ -111,7 +132,7 @@ Until April 2019, for about one and a half years as a whole and about a half-yea
...
+Shit happened 3 weeks after the `core-js@3` release. One April night, at 3 AM, I was driving home. Two deadly drunk 18-years-old girls in dark clothes decided somehow *to crawl* across a poorly lit highway - one of them lay down on the road, another sat down and dragged the first, but not from the road - directly under my wheels. That's what the witnesses said. I had no any chance to see them. One more witness said that before the accident they were just jokingly fighting on the road. Nothing unusual, it's Russia. One of them died and another girl went to a hospital. However, even in this case, according to Russian arbitrage practice, if the driver is not a son of a deputy or someone like that, he almost always will be found guilty - he should see and anticipate everything, and a pedestrian owes nothing to anyone. I could end up in prison for a long time, IIRC later the prosecutor requested 7 years.
```
Comment (zloirock) [inline docs/2023-02-14-so-whats-next.md:135]: However, it could be copied to `README`.

**Comment by nalzok** [comment]
Updated!

**Comment by zloirock** [comment]
Thanks!

**Pull request merged**